### PR TITLE
Fix unclosed HTML tags

### PR DESCRIPTION
**Pull request by steverep**
## Proposed change

Fixes and starts linting for tags that aren't properly closed.  Please check the display because I just closed the tags.
- MQTT, Assist, and `ha-label` fixes were obvious (wrong closing tag name)
- For debug dev tool, I added the missing closer `div`, but the HTML seems dirty.  I don't think the `form` and `card-content` classes even exist, and the `div` I closed seems unnecessary.  Feel free to push adjustments.

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/developer-tools/debug/developer-tools-debug.ts [hunk 1]
```diff
@@ -21,10 +21,11 @@ class HaPanelDevDebug extends SubscribeMixin(LitElement) {
...
           <div class="card-content">
-                  <ha-debug-connection-row
-                    .hass=${this.hass}
-                    .narrow=${this.narrow}
-                  ></ha-debug-connection-row>
+            <ha-debug-connection-row
+              .hass=${this.hass}
+              .narrow=${this.narrow}
+            ></ha-debug-connection-row>
+          </div>
```
Comment (silamon) [inline src/panels/developer-tools/debug/developer-tools-debug.ts:28]: It's copy paste from the profile settings. The div should be closed before ha-debug-connection-row for the display to look ok. Card content class exists and adds padding and the form class can be removed.

File: src/panels/developer-tools/debug/developer-tools-debug.ts [hunk 2]
```diff
@@ -21,10 +21,11 @@ class HaPanelDevDebug extends SubscribeMixin(LitElement) {
           class="form"
         >
           <div class="card-content">
```
Comment (silamon) [inline src/panels/developer-tools/debug/developer-tools-debug.ts:23]:
```suggestion
```

File: src/panels/developer-tools/debug/developer-tools-debug.ts (hunk 1 above)
Comment (silamon) [inline src/panels/developer-tools/debug/developer-tools-debug.ts:28]:
```suggestion
          <ha-debug-connection-row
            .hass=${this.hass}
            .narrow=${this.narrow}
          ></ha-debug-connection-row>
```

**Review by silamon — COMMENTED** [review]
It's copy paste from the profile settings. The div should be closed before ha-debug-connection-row for the display to look ok. Card content class exists and adds padding and the form class can be removed.

**Comment by steverep** [comment]
Your comment doesn't seem to match your suggestion.  Sure the `content` class is adding padding, but the `card-content` class doesn't exist, and the inner `div` should be removed.

**Comment by steverep** [comment]
Duh... I forgot it was also targeted by `::slotted`.  Seems to look okay like this.  Cleaned up a bit.

debug tools page screenshot

**Comment by silamon** [comment]
Sorry for the confusing comment. It looks ok in your screenshot even though the div with 'card-content' class may not be required. Hence the suggestion to remove de 'card-content' div. It is suggested in 2 comments since the review cannot be made over deleted lines.

To explain my comment further: 
On the profile page it is used as if the setting row can be used outside of the 'card-content' class.  Encapsulating the setting row in the 'card-content' adds padding from the 'card-content' class and from the setting row which defines padding on their own. 

On mobile view it will display a border to the top of the setting row, then you see the padding amount of the setting row and the amount of the 'card-content' class. 

It's a very minor nitpick, since the functionality works and the display is fine.

**Comment by steverep** [comment]
Ah, okay got it now.  The page needs some intro text at the top anyway (what the profile page uses it for), so maybe let's keep it and add some context for users.

**Comment by steverep** [comment]
... or just remove it and save added the intro text for another day 😄

**Review by bramkragten — APPROVED** [review]

**Pull request merged**